app/src/main/java/ch/epfl/polychef/fragments/UserListFragment.java

```diff
@@ -63,7 +76,7 @@ public void onAttach(@NonNull Context context) {
         super.onAttach(context);
         Preconditions.checkArgument(context instanceof HomePage, "The favourite miniature fragment wasn't attached properly!");
         HomePage homePage = (HomePage) context;
-        imageStorage = homePage.getImageStorage();
+        ImageStorage imageStorage = homePage.getImageStorage();
```

Comment:
If `imageStorage` is local to `onAttach` then we can just remove this line completely

Reply:
indeed !